caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1080,29 +1089,38 @@
               // re-enable all 'Product Types'
               // checkboxes
               $.each(
-                downloadForm.find('input.product_type_option_flag:checked'),
-                function () {
-                  $(this).prop('disabled', false)
-                }
+                  downloadForm.find('input.product_type_option_flag:checked'),
+                  function () {
+                    $(this).prop('disabled', false)
+                  }
               )
             }
           }
         })
 
         $('#results_bookmark').click(
-          function (event) {
-            event.preventDefault()
-            this._setBookmarkURL(new cadc.web.util.URI(event.target.href))
-            $('#bookmark_link').modal('show')
-          }.bind(this)
+            function (event) {
+              event.preventDefault()
+              this._setBookmarkURL(new cadc.web.util.URI(event.target.href))
+              $('#bookmark_link').modal('show')
+            }.bind(this)
         )
 
-        resultsVOTV.setDisplayColumns([])
+        resultsVOTV.setDisplayColumns(prevDisplayedColumns)
+        resultsVOTV.setUpdatedColumnSelects(prevColumnSelects)
 
-        // Set the default columns.
+        // Set the default columns and units.
         this._setDefaultColumns(resultsVOTV)
         this._setDefaultUnitTypes(resultsVOTV)
 
+        // Set default sort column and direction.
+        if (prevSortOptions['sortcol']) {
```

Comment:
This the same as `if (prevSortOptions.hasOwnProperty('sortcol')) {`, which is faster.

Reply:
Updated.